plugin/al/usdmayautils/AL/usdmaya/utils/DiffPrimVar.cpp

```diff
@@ -14,7 +14,8 @@
 // limitations under the License.
 //
 #include "AL/usdmaya/utils/DiffPrimVar.h"
-#include "AL/usd/utils/SIMD.h"
+#include <MayaUsdUtils/SIMD.h>
+#include <MayaUsdUtils/DiffCore.h>
```

Comment:
Why this new include is needed?

Reply:
There are calls to "vec3AreAllTheSame" and "compareArray" in this file from DiffCore and this header is indeed needed.

Previously DiffCore.h was getting included twice both in the *.h and *.cpp. Hence, including DiffPrimVar.h unintentionally would bring in DiffCore.h which is not good.